app/views/user_dashboard/show.html.erb

```diff
@@ -1 +1,68 @@
-<%= link_to "Log out", destroy_user_session_path, data: { turbo_method: :delete } %>
+<div id="user-dashboard">
+
+  <div class="navbar">
+    <div class="logo">
+      <%= vite_image_tag "images/logos/logo_text_white.png" %>
+    </div>
+
+    <div class="links">
+      <nav>
+        <a href="#">About</a>
+        <span>✾</span>
+        <a href="#">Sessions</a>
+        <span>✾</span>
+        <a href="#">Contact</a>
+        <span>✾</span>
+
+        <% if current_user %>
+          <%= link_to "Account", dashboard_path %>
+        <% else %>
+          <%= link_to "Log in", new_user_session_path %>
+        <% end %>
+      </nav>
+    </div>
+      <div class="log-out">
+        <%= link_to "Log out", destroy_user_session_path, data: { turbo_method: :delete } %>
+      </div>
+  </div>
+
+  <div class="user-dashboard-container">
+    <div class="user-profile-container">
+      <%= vite_image_tag "images/illustrations/33.png", id: "top", class: "leaf" %>
+      <%
+=begin%>
+ <%= vite_image_tag "images/illustrations/33.png", id: "bottom", class: "leaf" %>
+<%
+=end%>
+      <div class="user-profile-info">
+        <h2>Hello, <%= current_user.first_name %></h2>
+        <%= vite_image_tag "images/illustrations/yogi.svg", class: "profile-icon" %>
+        <p>10 classes attended</p>
+      </div>
+    </div>
+
+    <div class="user-options-container">
+      <div class="user-options-boxes">
+        <div class="user-options-info box-top">
+          <%= vite_image_tag "images/illustrations/calendar-icon.svg", class: "options-icon" %>
+          <p>Upcoming Classes</p>
+        </div>
+        <div class="user-options-info box-top">
+          <%= vite_image_tag "images/illustrations/history-icon.svg", class: "options-icon" %>
+          <p>Past Bookings</p>
+        </div>
+        <div class="user-options-info box-bottom">
+          <%= vite_image_tag "images/illustrations/edit-icon.svg", class: "options-icon" %>
+          <p>Edit Profile</p>
+        </div>
+        <div class="user-options-info box-bottom">
+          <%= vite_image_tag "images/illustrations/settings-icon.svg", class: "options-icon" %>
+          <p>Account Settings</p>
+        </div>
+      </div>
+    </div>
```

Comment:
You shouldn't be needing either the user-options-container or the user-options-boxes.
These two can be a single div